src/safariservices.cs

```diff
@@ -156,6 +156,7 @@ interface SFSafariPage
 
 	[Mac (10,12)]
 	[Protocol]
+	[Model]
```

Comment:
@dalexsoto see https://trello.com/c/9JANewiM/607-apiprotocoltest-generalcase-assumes-every-protocol-has-model

I can shut the test up with an exception or add [Model]. I wasn't sure what was worse.